### ISSUE TITLE
Key: image/class.  Can't parse serialized Example.

### ISSUE THREAD

**Issue by ghost** [codeaway23/number-plate-detection#3]
INFO:tensorflow:Starting Session.
I1223 18:10:23.516401 140558669547392 learning.py:754] Starting Session.
INFO:tensorflow:Saving checkpoint to path /content/aocr-logs/model.ckpt
I1223 18:10:23.721935 140554274965248 supervisor.py:1117] Saving checkpoint to path /content/aocr-logs/model.ckpt
INFO:tensorflow:Starting Queues.
I1223 18:10:23.727193 140558669547392 learning.py:768] Starting Queues.
INFO:tensorflow:global_step/sec: 0
I1223 18:10:30.392521 140554266572544 supervisor.py:1099] global_step/sec: 0
2019-12-23 18:10:36.825400: W tensorflow/core/framework/op_kernel.cc:1651] OP_REQUIRES failed at example_parsing_ops.cc:240 : Invalid argument: Key: image/class.  Can't parse serialized Example.
2019-12-23 18:10:36.825561: W tensorflow/core/framework/op_kernel.cc:1651] OP_REQUIRES failed at example_parsing_ops.cc:240 : Invalid argument: Key: image/class.  Can't parse serialized Example.
2019-12-23 18:10:36.825581: W tensorflow/core/framework/op_kernel.cc:1651] OP_REQUIRES failed at example_parsing_ops.cc:240 : Invalid argument: Key: image/class.  Can't parse serialized Example.
2019-12-23 18:10:36.825996: W tensorflow/core/framework/op_kernel.cc:1651] OP_REQUIRES failed at example_parsing_ops.cc:240 : Invalid argument: Key: image/class.  Can't parse serialized Example.
INFO:tensorflow:Error reported to Coordinator: <class 'tensorflow.python.framework.errors_impl.InvalidArgumentError'>, Key: image/class.  Can't parse serialized Example.
	 [[{{node ParseSingleExample/ParseSingleExample}}]]
2019-12-23 18:10:36.826406: W tensorflow/core/framework/op_kernel.cc:1651] OP_REQUIRES failed at example_parsing_ops.cc:240 : Invalid argument: Key: image/class.  Can't parse serialized Example.
I1223 18:10:36.826037 140554241394432 coordinator.py:224] Error reported to Coordinator: <class 'tensorflow.python.framework.errors_impl.InvalidArgumentError'>, Key: image/class.  Can't parse serialized Example.
	 [[{{node ParseSingleExample/ParseSingleExample}}]]
2019-12-23 18:10:36.826698: W tensorflow/core/framework/op_kernel.cc:1651] OP_REQUIRES failed at example_parsing_ops.cc:240 : Invalid argument: Key: image/class.  Can't parse serialized Example.
INFO:tensorflow:Caught OutOfRangeError. Stopping Training. RandomShuffleQueue '_0_shuffle_batch/random_shuffle_queue' is closed and has insufficient elements (requested 32, current size 0)
	 [[node shuffle_batch (defined at /usr/local/lib/python3.6/dist-packages/tensorflow_core/python/framework/ops.py:1748) ]]

**Comment by codeaway23** [comment]
maybe there's a problem with your tfrecords. can you tell me how to reproduce the error?

**Comment by ghost** [comment]
yeah I'm using trgd library to generate the text recognition dataset. then i use your code for converting into tfrecords. I have just added few more symbols in charset and updated the numberplates.py accordingly. but on training it gives me this error

**Comment by ghost** [comment]
> maybe there's a problem with your tfrecords. can you tell me how to reproduce the error?

let me know if you need more information

**Comment by Ram-Godavarthi** [comment]
@ksk-2811 This is a problem with tensorflow version i guess. I also got this error. but I changed the tensorflow version and tried. it worked out pretty well.

Try to install version 1.4.1

`pip3 install tensorflow==1.4.1`

**Comment by KnollFrank** [comment]
Please assert that in get_tf_records.py the variable MAX_STR_LEN has the same value as the property max_sequence_length in number_plates.py. Also assert that the variable named null in get_tf_records.py has the same value as the \<nul\> entry in charset-labels.txt.
